network/message/authorization.go

```diff
@@ -80,8 +80,9 @@ func initializeMessageAuthConfigsMap() {
 			return new(messages.SyncResponse)
 		},
 		Config: map[channels.Channel]flow.RoleList{
-			channels.SyncCommittee:     {flow.RoleConsensus},
-			channels.SyncClusterPrefix: {flow.RoleCollection},
+			channels.PublicSyncCommittee: {flow.RoleAccess},
```

Comment:
I didn't think we authorized messages sent on the public channels

Reply:
https://github.com/onflow/flow-go/pull/2832/commits/e9e60b7a32ac069fda2275071f9c575e4b0e28df